_overviews/scala3-book/domain-modeling-tools.md

```diff
@@ -21,56 +22,108 @@ Scala 3 provides many different constructs so we can model the world around us:
 
 This section briefly introduces each of these language features.
 
-
 ## Classes
 
 As with other languages, a _class_ in Scala is a template for the creation of object instances.
 Here are some examples of classes:
 
+{% tabs class_1 class=tabs-scala-version %}
```

Comment:
`class=tabs-scala-version` should not be used for "Scala 2 and 3" tabs. Soon I am adding assertions to make CI fail for this
```suggestion
{% tabs class_1 %}
```